assets/pages/paper-subscription-landing/components/content.jsx

```diff
@@ -41,6 +42,12 @@ const ContentHelpBlock = ({ faqLink, telephoneLink }: {faqLink: Element<string>,
   </ProductPageContentBlock>
 );
 
+function flashSaleAwareProductPageInfoChip(): string {
```

Comment:
Justin recently made quite a lot of changes to move all of the copy for flash sales into flashSale.js. We should probably do that with this copy to keep it consistent

Reply:
Also I think you could just call this function `getInfoChipCopy` to make it more consistent with other files where we do the same thing

Reply:
Yes, good point re: the naming. I will then leave this in `content.jsx` (for now). 

I thought about putting this InfoChip copy in flashSale.js, but since this is the first print flash sale since the new print page, I'm not sure if this is a fundamental part of print flash sale changes that will happen each time yet, and it's also not a change that happens for each product (like e.g. a headline change is).